accessibility/README.md

```diff
@@ -8,26 +8,30 @@ Web accessibility, [according to the W3C](https://www.w3.org/WAI/intro/accessibi
+The EU is currently in the process of formulating a robust and wide-ranging [EU Accessibility Act](http://ec.europa.eu/social/main.jsp?catId=1202) (EAA) which covers accessibility of a wide range of hardware devices, as well as e-commerce Web sites and other digital products. This is in addition to the [Web and Mobile Accessibility Directive](https://ec.europa.eu/digital-single-market/en/web-accessibility) which requires the accessibility of public sector Web sites and apps across Europe. The UK has been deeply involved in the development of the EAA, no doubt partly thanks to the emphasis placed on accessibility by the [Government Digital Service](https://accessibility.blog.gov.uk/) which has done exemplary work in this area.
 
-This may be due more to ignorance more than malice, but either way Web Matters will help educate its members and the wider Web community about our collective responsibility to keep the Web available for all. Accessibility is about making sure your service can be used by as many people as possible, which can only be a good thing for organisations.
+Likewise, many commercial organisations now realise that accessibility is something to be considered a high priority. They are working with experts, referencing the masses of [excellent information online](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction), and some are recruiting people with disabilities to help test Web sites. All this is commendable, however there are still too many organisations - and individual people working on the web - for whom accessibility is not seen as important; possibly due more to ignorance than malice.
 
 ## The Web Matters position
 
 Web Matters firmly agrees with Tim Berners-Lee, director of the [World Wide Web Consortium](https://www.w3.org) and inventor of the Web, [when he says](https://www.w3.org/standards/webdesign/accessibility):
 
 > The power of the Web is in its universality. Access by everyone regardless of disability is an essential aspect.
 
-We applaud [legislative efforts](https://www.gov.uk/guidance/equality-act-2010-guidance) which stress that organisations must make their Web sites accessible. However this by itself is not enough, and Web Matters is committed to educating its members about the importance of accessibility - not just because it's a legal requirement to do so, but because it is morally the right thing to do.
\ No newline at end of file
+We applaud [legislative](https://www.gov.uk/guidance/equality-act-2010-guidance) [efforts](http://ec.europa.eu/social/main.jsp?catId=1202) which stress that organisations must make their Web sites accessible. However this by itself is not enough, and Web Matters is committed to educating its members about the importance of accessibility - not just because it's a legal requirement to do so, but because it is morally the right thing to do.
+
+## Further reading
```

Comment:
Some more suggested resources:
https://www.coolfields.co.uk/2010/11/web-accessibility-types-of-disability/
https://www.w3.org/standards/webdesign/accessibility
https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction